Jenkinsfile

```diff
@@ -87,6 +87,10 @@ pipeline {
       steps {
         withGithubNotify(context: "Lint") {
           stageStatusCache(id: 'Lint'){
+            // test the ./dev-tools/run_with_go_ver used by the Unified Release process
+            dir("${BASE_DIR}") {
+              sh "HOME=${WORKSPACE} GO_VERSION=${GO_VERSION} ./dev-tools/run_with_go_ver make test-mage"
+            }
```

Comment:
This works 

<img width="1257" alt="image" src="https://user-images.githubusercontent.com/2871786/190223591-ac218f82-7664-40ed-8435-16274ccc1e1a.png">